docs/src/extensions.md

```diff
@@ -51,6 +51,14 @@ The [`@constraint`](@ref) macro always calls the same three functions:
 Adding methods to these functions is the recommended way to extend the
 [`@constraint`](@ref) macro.
 
+### Adding `parse_constraint` methods
+
+TODO
```

Comment:
Please avoid leaving visible TODOs in the documentation now that we're preparing for the release. You can use the `@meta` tag and say
```
# TODO(Benoit): Introduce the `set_to_sense` docstring.
```
instead (e.g., at the top of https://raw.githubusercontent.com/JuliaOpt/JuMP.jl/master/docs/src/index.md).

Reply:
Good point, I forgot about this